docker/actinia-core-alpine/Dockerfile

```diff
@@ -1,8 +1,10 @@
+FROM mundialis/actinia:alpine-dependencies-v1 as build-base
 FROM mundialis/grass-py3-pdal:releasebranch_8_0-alpine as grass
```

Comment:
Would it make sense to go directly to grass 8.2 here?

Reply:
yes! I updated it